contrib/metrics/grafana/dashboards/zcashd-metrics.json

```diff
@@ -31,7 +31,7 @@
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "T9XAoUn4k"
+        "uid": "${DS_PROMETHEUS}"
```

Comment:
This does not match the `uid` in `contrib/metrics/grafana/provisioning/datasources/prometheus.yaml`, which is what it needs to match for the instructions in https://zcash.github.io/zcash/user/metrics.html#metrics-collection-with-docker to work.

Additionally, I did some searching online for `${DS_PROMETHEUS}` and couldn't find any guidance on how to use it properly. The few pages I found mentioning it said "manually replace it with the actual `uid` from your datasource", which seems no better than the status quo for users importing the dashboard.

Reply:
Good find and added commit. Regardless this shouldn't be hardcoded into anything as it breaks exports. It appears this was commited without using the allow external usage on export. As it stands now, using this dashboard requires a user to sanity check the export UID on host and then find and replace all these hardcoded UIDs in the json then import. That is not correct and these should bind into an imported dashboard gracefully

Reply:
I still cannot find documentation supporting `${DS_PROMETHEUS}` as being a variable that Grafana understands and that will behave correctly when running it via the `docker-compose.yml` pathway. Can you provide a relevant link? It is not sufficient for this file to only be used via the Grafana GUI's "import dashboard" pathway.

Reply:
I haven't used uid so maybe it is something specific to the WSL implementation. Perhaps `uid: DS_PROMETHEUS` would work instead. I think maybe there was an issue when exporting the json dashboard here to github. Do you recall if folks selected the external link option here: https://github.com/grafana/grafana/issues/60769#issuecomment-1366913837 ? I see within this environment it might work, but if someone were to take the json dashboard here and import it into grafana like normal, it will not work because all the UIDs are hardcoded to "T9XAoUn4k"